### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by tkwatson63** [bountysource/core#1491]
I am visually impaired and use a screen reader to navigate my computer and the web. Yesterday I attempted to delete an integration from HA and was surprised to find that it was not possible anymore. I asked for sighted assistance and found that the icon to delete an integration was not accessible by my screen reader. My daughter clicked on the icon and then I was able to remove it. Sort of worries me about what else is not "visible" to those of us who cannot see a monitor. If you could look into this it would be much appreciated.
